.ci/pitest.sh

```diff
@@ -44,22 +44,22 @@ pitest-annotation|pitest-design \
 |pitest-tree-walker \
 |pitest-utils \
 |pitest-java-ast-visitor)
-  mvn --no-transfer-progress -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
+  mvn --no-transfer-progress -e -P$1 clean org.pitest:pitest-maven:mutationCoverage;
```

Comment:
We need test phase for maven

Reply:
Reason is shown in my question to pitest. I restored it for now, but we will execute tests twice now. Once naturally because of test phase, and 2nd when pitest checks our green suite and starts its process on mutations.

Reply:
Example:
Pitest run before upgrade: https://github.com/checkstyle/checkstyle/runs/5075559686?check_suite_focus=true#step:4:1111
> [INFO] Total time:  58.483 s

Pitest run after upgrade: https://github.com/checkstyle/checkstyle/runs/5075544886?check_suite_focus=true#step:4:1780
> [INFO] Total time:  04:45 min

CI run time increased by 4.9 times

Reply:
We can ignore this version why discussion is going on in pitest issue